apis/visibility/v1alpha1/types.go

```diff
@@ -32,13 +32,13 @@ type PendingWorkload struct {
 	Priority int32 `json:"priority"`
 
 	// LocalQueueName indicates the name of the LocalQueue the workload is submitted to
-	LocalQueueName string `json:"localQueueName"`
```

Comment:
I don't think we want this change, it might be due to resolving?

Reply:
Good point